osgar/drivers/lora.py

```diff
@@ -113,6 +113,20 @@ def send_data(self, data):
         self.last_transmit = self.publish('raw', data + b'\n')
         return self.last_transmit
 
+    def on_radio(self, data):
```

Comment:
I am somewhat confused. How can this work? What happens in `run` when there is no lora device communicating over 'raw'?

Reply:
one possibility is to copy & paste lora code to some "virtual lora", which does not use this low level parsing of `raw` streams and has directly "packets" received from `radio`??

Reply:
Yeah, probably some kind of refactoring could help me understand better. And it feels wrong to deal with artifacts in 'osgar/drivers/'. Or we can merge "as is" and move on but then we risk this code will be forever yours as everyone will be afraid to touch it due to lack of understanding.